src/services/insights/readme.md

```diff
@@ -0,0 +1,97 @@
+# Insights API
```

Comment:
use swagger instead

Reply:
The readme will be removed after the frontend is 100% up and running, as J finds it more helpful than Swagger at the moment.